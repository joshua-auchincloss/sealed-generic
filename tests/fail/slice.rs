use sealed_generic::SealedGeneric;

#[derive(SealedGeneric)]
#[define(
    sealed,
    types(ty = i32),
)]
pub struct NoSlices<T: SealedSomeGeneric> {
    value: [T; 4],
}

fn main() {}
