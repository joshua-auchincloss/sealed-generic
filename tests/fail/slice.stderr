error: proc-macro derive panicked
 --> tests/fail/slice.rs:3:10
  |
3 | #[derive(SealedGeneric)]
  |          ^^^^^^^^^^^^^
  |
  = help: message: unknown type

error[E0405]: cannot find trait `SealedSomeGeneric` in this scope
 --> tests/fail/slice.rs:8:24
  |
8 | pub struct NoSlices<T: SealedSomeGeneric> {
  |                        ^^^^^^^^^^^^^^^^^ not found in this scope
